app/serializers/base_serializer.rb

```diff
@@ -1,5 +1,6 @@
 # The serializer which all serializers inherit from.
 class BaseSerializer
+  include Rails.application.routes.url_helpers
```

Comment:
I'd love some white space, no biggie though

Reply:
Added white space below but house says not to white space above